README.md

```diff
@@ -310,13 +310,11 @@ They can be added to a scenario file like this:
         apply-build-script-change-to = "build.gradle.kts"
         apply-project-dependency-change-to {
             files = ["build.gradle"]
-            // Example: This will apply a unique combination of 5 projects from a set of 10 projects.
-            // Number of combinations can be calculated by typing "10 choosen 5" in your favorite search engine.
-            // When both values are equal there is just one combination possible, so all
-            // runs will have the same combination of the projects applied as dependency.
-            // Default values are 10 and 5. 
-            projects-set-size = 10
-            applied-projects-set-size = 5
+            # Default number of applied-projects-count is 3.
+            # Gradle-profiler will generate minimal number of projects to have a unique combination of project dependencies every run.
```

Comment:
I think this is not easy to understand...

Reply:
How about:
```suggestion
            # Gradle Profiler will simulate changes to project dependencies by generate some additional projects and then add a combination of project dependencies to every non-generated subprojects before each iteration.
            # The profiler will generate the minimal number of subprojects to allow for a unique combination of dependencies to be used for each iteration.
```